docs/faq.rst

```diff
@@ -258,8 +258,10 @@ https://github.com/nabilm/ansible-splash.
 
 Website is not rendered correctly
 ---------------------------------
+In this case we have two types of issues: wrong rendering and Splash crashing. 
```

Comment:
What do you think about splitting this in two sections - "Website is not rendered correctly" and "Splash is crashing"? currently both crashing and wrong rendering is under "wrong rendering"